steps/appcenter-app-release/1.0.4/step.yml

```diff
@@ -0,0 +1,84 @@
+title: AppCenter App Release
+summary: |
+  Release an application on Microsoft AppCenter
+description: |
+  Release an application on Microsoft AppCenter
+website: https://github.com/fileformat/bitrise-step-appcenter-app-release
+source_code_url: https://github.com/fileformat/bitrise-step-appcenter-app-release
+support_url: https://github.com/fileformat/bitrise-step-appcenter-app-release/issues
+published_at: 2019-11-14T14:29:07.683519-05:00
+source:
+  git: https://github.com/fileformat/bitrise-step-appcenter-app-release.git
+  commit: 53de7d586a5118ba8931c7d72f87fdfd059ec627
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  - name: curl
+  apt_get:
+  - name: jq
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- appcenter_api_token: $APPCENTER_API_TOKEN
+  opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: AppCenter API Token
+- appcenter_name: $APPCENTER_NAME
+  opts:
+    is_expand: true
+    is_required: true
+    title: AppCenter Application Name
+- appcenter_org: $APPCENTER_ORG
+  opts:
+    is_expand: true
+    is_required: true
+    title: AppCenter Organization
+- artifact_path: $ARTIFACT_PATH
+  opts:
+    is_expand: true
+    is_required: true
+    title: path to the built application file (.ipa or .apk)
+- distribution_groups: $DISTRIBUTION_GROUPS
+  opts:
+    description: Groups should be comma separated e.g. group1, group2. If not specified,
```

Comment:
It is recommended to use `|` separated lists in inputs: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#inputs-which-can-accept-a-list-of-values

Reply:
Might be worth keeping as-is though, for backwards-compatibility: https://github.com/bitrise-io/bitrise-steplib/pull/2156/files#diff-9192b93360796b42e3c382675e0e5914R56

Reply:
Will not fix: do not want to break backward compatibility.  I don't plan on make a 2.x (or 1.1.x) version, since bitrise is going to do their own version.